src/cmd-buildextend-live

```diff
@@ -801,7 +862,21 @@ with open(build_semaphore, 'w') as f:
     f.write(f"{time.time_ns()}")
 
 try:
-    generate_iso()
+    if COSA_USE_LIVEISO is None:
+        generate_iso()
+    else:
+        command = [
+            "/usr/bin/cosa", "supermin-run",
+            "--cache",  "/usr/lib/coreos-assembler/runvm-osbuild",
+            "--config",
+            image_for_disk_json,
+            "--mpp",
+            f"/usr/lib/coreos-assembler/osbuild-manifests/coreos.osbuild.{basearch}.mpp.yaml",
+            "--filepath",
+            tmpisofile
+        ]
+        subprocess.run(command)
```

Comment:
We'll need a call to `coreos-installer iso extract pxe` here to extract the other artifacts from the live ISO until we rework `runvm-osbuild` and the osbuild manifests to support outputting all of them upfront as discussed.